aea/aea_builder.py

```diff
@@ -970,6 +965,7 @@ def add_component_instance(self, component: Component) -> "AEABuilder":
         You will have to `reset()` the builder before calling `build()` again.
 
         :param component: Component instance already initialized.
+        :return: self
```

Comment:
not consistent with `:return: the AEABuilder.` below

Reply:
(half the file uses one, half the other :D)...keeping as is now